src/jdk.sctp/share/classes/sun/nio/ch/sctp/SctpStdSocketOption.java

```diff
@@ -27,7 +27,9 @@
 import com.sun.nio.sctp.SctpSocketOption;
 import java.lang.annotation.Native;
 
-public class SctpStdSocketOption<T>
+public record SctpStdSocketOption<T>(String name,
+                                     Class<T> type,
+                                     int constValue)
```

Comment:
This changes the implementation of `toString`, `equals` and `hashCode`. Have you verified that it is OK to do so?